🐘 Update custom-chains.mdx (#575)
TrueFiEng/useDApp: packages/docs/docs/tutorials/custom-chains.mdx

  getExplorerAddressLink: (address: string) => `https://tutorialchain.etherscan.io/address/${address}`,
  getExplorerTransactionLink: (transactionHash: string) => `https://tutorialchain.etherscan.io/tx/${transactionHash}`,
}

> // IMPORTANT: fill these object with your own constants

Reviewer: ```suggestion
// IMPORTANT: Fill that object with your own data.
```
